src/main/groovy/com/github/eerohele/SaxonXsltTask.groovy

```diff
@@ -175,459 +187,459 @@ class SaxonXsltTask extends DefaultTask {
     // START ADVANCED OPTIONS
 
     void allowExternalFunctions(Boolean x) {
-        this.advancedOptions['allow-external-functions'] = x;
```

Comment:
I understand these semicolons are redundant, but if we want to remove it, we should definitely do it in a separate commit.

Reply:
I see this makes it difficult to review, sorry about that.